### DISCUSSION TITLE
Config will not complete

### DISCUSSION THREAD

**Discussion by MatzeLoCal** [flobz/psa_car_controller#208] [Q&A] [answered]
Hi,

I try to install psa_car_controller on a Raspberry Pi Zero. 
Everything works fine until I get past the point where I enter the values in "My car info config"
When I hit "Submit" and wait I'll get "Success !!! Go to otp config" but as soon as I click the link I'll get back to the first config-page.

Here is the console output:
`$ python3 server.py -f test.json -c charge_config1.json -l 192.168.188.133 --web-conf
2021-08-26 13:01:35,783 :: INFO :: update_data
 * Running on http://192.168.188.133:5000/ (Press CTRL+C to quit)
192.168.188.26 - - [26/Aug/2021 13:02:13] "GET /config HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:02:13] "GET /assets/clientside.js?m=1629978516.2198923 HTTP/1.1" 304 -
192.168.188.26 - - [26/Aug/2021 13:02:14] "GET /_dash-layout HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:02:14] "GET /_dash-dependencies HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:02:14] "POST /_dash-update-component HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:02:14] "POST /_dash-update-component HTTP/1.1" 200 -
Requested API level 29 is larger than maximum we have, returning API level 28 instead.
2021-08-26 13:06:25,127 :: INFO :: save config change
2021-08-26 13:06:26,724 :: INFO :: 
Your vehicles: [CAR_JSON]
2021-08-26 13:06:26,741 :: INFO :: save config change
192.168.188.26 - - [26/Aug/2021 13:06:28] "POST /_dash-update-component HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:28] "GET /config_otp HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "GET /assets/clientside.js?m=1629915130.6177464 HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "GET /assets/clientside.js?m=1629978516.2198923 HTTP/1.1" 304 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "GET /_dash-layout HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "GET /_dash-dependencies HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "POST /_dash-update-component HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "GET /config HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "GET /assets/clientside.js?m=1629978516.2198923 HTTP/1.1" 304 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "GET /_dash-layout HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:29] "GET /_dash-dependencies HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:30] "POST /_dash-update-component HTTP/1.1" 200 -
192.168.188.26 - - [26/Aug/2021 13:08:30] "POST /_dash-update-component HTTP/1.1" 200 -`

The values in CAR_JSON seems to be alright as it's the same data I get in my Swift Framework (which I'm currently develop for my own app)
<img width="233" alt="Screenshot 2021-08-26 at 13 46 02" src="https://user-images.githubusercontent.com/5370895/130961409-b5e48f9d-567e-46de-9db0-40607b5c328e.png">

**Comment by kurim** [comment, marked as answer]
Did you try this: https://github.com/flobz/psa_car_controller/issues/181#issuecomment-878534482
It seems a bit buggy on change to otp config
